frontend/src/routes/LandingPage/index.js

```diff
@@ -140,6 +141,9 @@ class LandingPage extends Component {
             </li>
           )}
         </LinkWrapper>
+        {admission.is_open && (
```

Comment:
Shouldn't this be when the admission is _not_ open

Reply:
Yes, thats true